pages/index.vue

```diff
@@ -1,4 +1,57 @@
+<script lang="ts" setup>
```

Comment:
Should always be `<script setup lang="ts">`. Doesn't change anything but this is perhaps the most common vue convention. The syntactic sugar is actually called "script setup".

Same for all instances.

Reply:
Done!